static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -178,7 +184,7 @@ class GroupDetails extends Component<Props, State> {
       error_has_replay: Boolean(event?.tags?.find(({key}) => key === 'replayId')),
       group_has_replay: Boolean(group?.tags?.find(({key}) => key === 'replayId')),
       num_comments: group ? group.numComments : -1,
-      project_has_replay: group?.project.hasReplays,
+      project_has_replay: project.hasReplays,
       project_has_minified_stack_trace: group?.project.hasMinifiedStackTrace,
```

Comment:
@ryan953 can you update this line of code as well? So we don't read off `group.project` here anymore